src/codegen/PRODUCT_TAGS.php

```diff
@@ -2,12 +2,12 @@
 /**
  * This file is generated. Do not modify it manually!
  *
- * @generated SignedSource<<37b77a78e65250c03eccfb679201050c>>
+ * @generated SignedSource<<9273e4ec8cb5789cc665443973de0f65>>
  */
 namespace HHVM\UserDocumentation;
 
 const dict<APIProduct, string> PRODUCT_TAGS = dict[
-  APIProduct::HACK => 'HHVM-4.102.0',
-  APIProduct::HSL => 'v4.94.0',
-  APIProduct::HSL_EXPERIMENTAL => 'v4.99.0',
+  APIProduct::HACK => 'HHVM-4.108.0',
+  APIProduct::HSL => 'v4.108.1',
```

Comment:
Are things like 'HH\Lib\Vec\map()` still listed under HSL API reference after this bump?